nightingale_navigation/configs/amcl.yaml

```diff
@@ -0,0 +1,52 @@
+#
+# http://www.ros.org/wiki/amcl
+# The first commented value is the default per the wiki
+# The second commented value is the default per nav/amcl/examples
+# The third commented value is the defaults in amcl_node.cpp
+#
+# The map file is specified in launch/amcl.launch
+
+odom_frame_id:    odom
+base_frame_id:    base_link
+global_frame_id:  map
+  
+# Overall filter parameters
+min_particles:        350       #default 100 ,          last 100
+max_particles:        2000      #default 5000
+kld_err:              0.01      #default 0.01,          last 0.05
+kld_z:                0.99      #default 0.99,          last 0.95
+update_min_d:         0.2       #default 0.2,           last 0.1
+update_min_a:         0.5235    #default pi/6 (0.5235)  last 0.25
+resample_interval:    1         #default 1              last 2
+transform_tolerance:  0.5       #default 0.1            last 0,1, before 1.0
+recovery_alpha_slow:  0.001     #default 0.0
+recovery_alpha_fast:  0.1       #default 0.0
+initial_pose_x:       0.0       #default 0.0
+initial_pose_y:       0.0       #default 0.0
+initial_pose_a:       0.0       #default 0.0
+initial_cov_xx:       0.2      #default 0.25
+initial_cov_yy:       0.2       #default 0.25
+initial_cov_aa:       0.12    #default (pi/12)*(pi/12) (0.06853)
+gui_publish_rate:     -1.0      #default -1.0
+save_pose_rate:       0.5       #default 0.5
+use_map_topic:        false     #default false
+first_map_only:       false     #default false
+
+# Laser Model Parameters 
+laser_min_range:            -1.0    #default -1.0
+laser_max_range:            -1.0    #default -1.0
+laser_max_beams:            60     #default 30
+laser_z_hit:                0.90   #default 0.95
```

Comment:
Are you able to explain the reasoning of why some parameters are changed from their defaults?

Reply:
so i took these params from https://github.com/nightingale-project/kinova-movo/blob/master/movo_demos/config/2d_nav/amcl.yaml

Reply:
ive updated the params and now theyre different from what kinova uses. here's a summary:
max_particles increased from 2000 to 5000. I think we have extra compute to spare
laser min and max ranges now uses the actual values i found for our SIC laser
The rest of the laser model params are what kinova uses. See page 171 of Probabilistic Robotics for an explanation of the params. We use the likelihood_field_model
I also updated the motion model to use 'omni-corrected' instead of 'omni' since the latter had a [bug](https://github.com/ros-planning/navigation/issues/20). i also had to update those alpha constants as per [this post](https://answers.ros.org/question/227811/tuning-amcls-diff-corrected-and-omni-corrected-odom-models/)